composer.json

```diff
@@ -10,7 +10,7 @@
         "composer/semver": "^3.3",
         "composer/xdebug-handler": "^3.0",
         "cweagans/composer-patches": "^1.7",
-        "friendsofphp/php-cs-fixer": "^3.8",
+        "friendsofphp/php-cs-fixer": "3.8.0",
```

Comment:
What is the issue here? The version cannot be locked, but rather solved

Reply:
See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6470

Reply:
Seems it was fixed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6473
Let's wait for release then, so we avoid such version lock.

Reply:
that patch exists in 3.9.2 but the issue still persists for commented code indented, that's why the issue still opened.

Reply:
Updated to ^3.9.3 https://github.com/symplify/symplify/pull/4212/commits/b5343ceb06af7281e52f17c0360258d9d5f6c303

Reply:
it seems still error in 3.9.3:

```bash
There was 1 failure:
1) Symplify\EasyCodingStandard\Tests\Set\Psr12\Psr12Test::test with data set #0 (Symplify\SmartFileSystem\SmartFileInfo Object (...))
packages/easy-coding-standard/tests/Set/Psr12/Fixture/fixture2.php.inc
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
     $doSomething = true;\n
 }\n
 if ($expr1) {\n
-    // if body\n
+// if body\n
```

ref https://github.com/symplify/symplify/runs/7327330149?check_suite_focus=true#step:5:58

Reply:
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6478

Reply:
I see.

Reply:
Let's skip it for now, as the rest seems ok and this is direct php-cs-fixer impact.